### DISCUSSION TITLE
[Feature] Easy photo selection in search

### DISCUSSION THREAD

**Discussion by ivsafronov** [immich-app/immich#12887] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hi! Thanks for the great software.

I would like to recommend one improvement.
One of the today's problem is a large amount of useless images from phones. It's usually 5-15% of all photos ( it's now I'm clever and delete such things right away :) )

I'd like to be able, by typing “screenshot” in the search to select nearby photos in a row by holding **Shift**, as is done in the main feed, to also delete them with the **Del** key.

Also, it would be nice to sort the search output not only by match, but also, for example, by date. The search works fine, but it lacks a bit of flexibility.

Thank you, with respect!

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by ivsafronov** [comment]
I should add that this select mechanism is so user friendly that you can add it to memories and faces as well!
If I upload a new large set of photos, there are dozens of new unnecessary faces, which can also be hidden with one move

**Comment by bo0tzz** [comment]
#9159
